app/views/spree/admin/images/edit.turbo_stream.haml

```diff
@@ -3,7 +3,13 @@
     %h2= t(".title")
 
     -# Display image in the same way it appears in the shopfront popup
-    %p= image_tag @image.persisted? ? @image.url(:large) : Spree::Image.default_image_url(:large), width: 433, height: 433
+    - if defined?(@errors) && !@errors.empty?
+      - @errors.each do |error|
+        %p
+          = error
```

Comment:
FYI usually `errors.full_message.to_sentence` is enough for cases like this.

Reply:
The @errors object actually contains only the full messages.